### PR TITLE
Adding NeMo Guardrails template

### PR DESCRIPTION
**Pull request by Aditib2409**

### CODE REVIEW

File: templates/nvidia-guardrails-with-RAG/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+```
+export NVIDIA_API_KEY=<your nvidia_api_key>
+export NGC_API_KEY=<your ngc_api_key>
+```
+### NeMo Guardrails
+
+To add guardrails, first you need to clone the nemoguardrails package with langchain runnable integrations as follows:
+
+Create a directory named `nemoguardrails`, clone and install the package with all necessary dependencies
```
Comment (efriis) [inline templates/nvidia-guardrails-with-RAG/README.md:35]: This is a bit heavyweight for a template

File: templates/nvidia-guardrails-with-RAG/README.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# Guardrails Integration with LangChain Expression Language Template
+
+This example demonstrates integrating NeMo Guardrails with LangChain Expression Language Templates. Here we start of with a `nvidia-rag-canonical` from the LangChain [templates](https://github.com/langchain-ai/langchain/tree/master/templates/nvidia-rag-canonical) which performs RAG using Milvus Vector Store and NVIDIA Models, both embedding and Chat.
```
Comment (efriis) [inline templates/nvidia-guardrails-with-RAG/README.md:3]: If people use this, they'll be starting from this template `nvidia-guardrails-with-rag` instead

**Review by efriis — COMMENTED** [review]
Hey there! Thanks for resubmitting with the extra files not included. I'm not sure NeMo guardrails in this form is a good fit for a template (having to `git clone` another repo isn't part of any other templates). Given the template is similar to `nvidia-rag-canonical`, maybe this is better as a blog post that walks people how to modify that starter template to include nemo guardrails?

Is there any way to install it without cloning a repo?

File: templates/nvidia-guardrails-with-RAG/README.md (hunk 1 above)
Comment (Aditib2409) [reply templates/nvidia-guardrails-with-RAG/README.md:35]: @efriis a simple 'pip install nemoguardrails' should work. All the latest modules needed here have been updated on the guardrails too. Please let me know how I can make this change in the README.md file

File: templates/nvidia-guardrails-with-RAG/README.md (hunk 2 above)
Comment (Aditib2409) [reply templates/nvidia-guardrails-with-RAG/README.md:3]: Yes. Got it! Apologies for the typo. Let me know how I can make the necessary changes

**Comment by efriis** [comment]
Closing on behalf of one of your teammates! Think a different use case will be posted later for features

**Pull request closed without merging**